src/sbt-test/interplay/library-with-plugin/test

```diff
@@ -7,26 +7,30 @@ $ exec git add .
 $ exec git commit -m commit
 
 # Setup remote git repo in target directory
-$ exec git init --bare target/remote
-$ exec git remote add origin target/remote
+$ exec git init --bare $PWD/target/remote
```

Comment:
Unrelated to your comment, but as far as cross building is concerned, this is where you need to update the tests. At the moment you're only testing that the `interplay` plugin itself can run against different sbt versions, but that's not really that important because the Play build will only use one version of sbt, either 0.13.6 or 1.0. What is important is that the configuration that `interplay` produces for cross building sbt plugins results in correct cross building, particularly when a release is done. If you look down lower in this file, you'll see this:

```
 > contains target/scala-2.11/publish-version no-publish:1.2.3
 > contains target/scala-2.12/publish-version no-publish:1.2.3
 > contains mock-sbt-plugin/target/scala-2.10/sbt-0.13/publish-version Bintray-Sbt-Publish-playframework-sbt-plugin-releases-mock:1.2.3
 > contains mock-library/target/scala-2.11/publish-version sonatype-staging:1.2.3
 > contains mock-library/target/scala-2.12/publish-version sonatype-staging:1.2.3
 > contains mock-sbt-library/target/scala-2.10/publish-version sonatype-staging:1.2.3
``` 

As you can see, it's verifying that the release did run for both Scala 2.11 and Scala 2.12 on runtime libraries, but it's only checking that sbt plugins and libraries were built against sbt 0.13 and Scala 2.10, it should be checking for sbt 1.0 and Scala 2.12 as well.

I'd recommend removing the cross building configuration from travis because it achieves nothing, and instead update these tests to ensure that the projects that use `interplay` cross build correctly.